src/pages/UserPage/UserPage.tsx

```diff
@@ -162,12 +162,9 @@ const UserPage = () => {
     };
   }, [contributionData]);
 
-  useMemo(() => {
+  useEffect(() => {
     const url = `${window.location.protocol}//${window.location.host}/${userName}`;
```

Comment:
could you verify whether it is working or not? please also add a video of the changes in the description